jwst/assign_wcs/assign_wcs.py

```diff
@@ -97,6 +97,13 @@ def _add_3rd_axis(input_model):
     """
     Add WCS keywords and their default values for the spectral axis.
 
+    Parameters
+    ----------
+    input_model: `~jwst.datamodels.DataModel`
```

Comment:
The numpy documentation for docstrings says that in parameter descriptions, a space is required before the colon.  I saw a comment on reddit that says the documentation won't be built correctly if the space is missing.  Is this true?  Do we really need a space both before and after the colon?  If so, there are a number of cases where the space is missing.

Reply:
I would think it only made a spacing difference, but we might as well be consistent.

Reply:
I think there needs to be a space. I have not tested it withut but have been told strictly that there should be a space.